tests/query_tests.rs

```diff
@@ -10,6 +10,7 @@ use test_case::test_case; // Parametrized tests
 #[test_case("histogram.cql", vec!["histogram.rs"]; "inconclusive - histogram")]
 #[test_case("request_size.cql", vec![]; "request_size")]
 #[test_case("request_size_avg.cql", vec![]; "inconclusive - request_size_avg")]
+#[test_case("return_trace.cql", vec![]; "inconclusive - return_trace")]
```

Comment:
I guess it is inconclusive because we do not have code generation for it yet.

Reply:
Yes, I made edits to codegen_envoy.rs in a way that I think will make it work.  I just don't want to trust that it works before we get the serialization thing figured out, because if it breaks on something simple like height it will absolutely break here.